### ISSUE TITLE
No red squiggles

### ISSUE THREAD

**Issue by karera123** [microsoft/vscode-cpptools#2114]
I am wondering why there's no red squiggles in syntax errors in c/c++ codes. I installed c/c++ extension v0.17.4, vscode v1.24.0 and I'm using bash as integrated terminal. This is my c_cpp_properties.json: 
```
{
    "configurations": [
        {
            "name": "WSL",
            "intelliSenseMode": "clang-x64",
            "compilerPath": "/usr/bin/gcc",
            "includePath": [
                "${workspaceFolder}",
                "/mnt/c/libraries/lib1/include",
                "C:/libraries/lib2/include",
                "/usr/include",
                "/usr/include/c++/7"
            ],
            "defines": [],
            "browse": {
                "path": [
                    "${workspaceFolder}",
                    "/mnt/c/libraries",
                    "/usr/include",
                    "/usr/include/c++/7"
                ],
                "limitSymbolsToIncludedHeaders": true,
                "databaseFilename": ""
            },
            "cStandard": "c11",
            "cppStandard": "c++17"
        }
    ],
    "version": 4
}
```

**Comment by sean-mcmanus** [comment]
When you open the file do you see a Microsoft.VSCode.CPP.IntelliSense.Msvc.exe process start up and then disappear and/or do you see messages about the IntelliSense client disconnecting in the C/C++ output log? Do squiggles appear in any file, such as with just "@" in it?

**Comment by karera123** [comment]
`Microsoft.VSCode.CPP.IntelliSense.Msvc.exe` crashes (even when I run it manually), all I see is `Microsoft.VSCode.CPP.Extension.exe` and C/C++ output log shows this:
```
initialized
workspace/didChangeConfiguration
IntelliSense Engine = Default.
The extension will use the Tag Parser for IntelliSense when #includes don't resolve.
Autocomplete is enabled.
Error squiggles are enabled.
File exclude: **/.git
File exclude: **/.svn
File exclude: **/.hg
File exclude: **/CVS
File exclude: **/.DS_Store
File exclude: **/.vscode
Search exclude: **/node_modules
Search exclude: **/bower_components
Search exclude: **/.vscode
textDocument/didOpen
cpptools/queryCompilerDefaults
Attempting to get defaults from compiler found on the machine: /usr/bin/gcc
Attempting to get defaults from compiler in "compilerPath" property: '/usr/bin/gcc'
cpptools/activeDocumentChange
cpptools/textEditorSelectionChange
cpptools/didChangeFolderSettings
Attempting to get defaults from compiler in "compilerPath" property: '/usr/bin/gcc'
Code browsing service initialized
Attempting to get defaults from compiler in "compilerPath" property: '/usr/bin/gcc'
  Folder: C:/Users/John Nikko/AppData/Local/Packages/CanonicalGroupLimited.UbuntuonWindows_79rhkp1fndgsc/LocalState/rootfs/usr/lib/gcc/x86_64-linux-gnu/7/include/ will be indexed
  Folder: C:/Users/John Nikko/AppData/Local/Packages/CanonicalGroupLimited.UbuntuonWindows_79rhkp1fndgsc/LocalState/rootfs/usr/local/include/ will be indexed
  Folder: C:/Users/John Nikko/AppData/Local/Packages/CanonicalGroupLimited.UbuntuonWindows_79rhkp1fndgsc/LocalState/rootfs/usr/lib/gcc/x86_64-linux-gnu/7/include-fixed/ will be indexed
  Folder: C:/Users/John Nikko/AppData/Local/Packages/CanonicalGroupLimited.UbuntuonWindows_79rhkp1fndgsc/LocalState/rootfs/usr/include/ will be indexed
  Folder: C:/Users/John Nikko/Desktop/codes/ will be indexed
Discovering files...
textDocument/codeAction
cpptools/didChangeFolderSettings
textDocument/didOpen
Attempting to get defaults from compiler in "compilerPath" property: '/usr/bin/gcc'
  Processing folder (recursive): C:/Users/John Nikko/AppData/Local/Packages/CanonicalGroupLimited.UbuntuonWindows_79rhkp1fndgsc/LocalState/rootfs/usr/lib/gcc/x86_64-linux-gnu/7/include/
Failed to spawn IntelliSense process: 
Shutting down IntelliSense server: C:\Users\John Nikko\Desktop\codes\c\data-structure\linked-list\link-list.cpp
Checking for syntax errors: file:///c%3A/Users/John%20Nikko/Desktop/codes/c/data-structure/linked-list/link-list.cpp
Attempting to get defaults from compiler in "compilerPath" property: '/usr/bin/gcc'
Failed to spawn IntelliSense process: 
Shutting down IntelliSense server: C:\Users\John Nikko\Desktop\codes\c\data-structure\linked-list\link-list.cpp
Failed to create IntelliSense client. Can't create intellisense client for C:\Users\John Nikko\Desktop\codes\c\data-structure\linked-list\link-list.cpp
  Processing folder (recursive): C:/Users/John Nikko/AppData/Local/Packages/CanonicalGroupLimited.UbuntuonWindows_79rhkp1fndgsc/LocalState/rootfs/usr/local/include/
  Processing folder (recursive): C:/Users/John Nikko/AppData/Local/Packages/CanonicalGroupLimited.UbuntuonWindows_79rhkp1fndgsc/LocalState/rootfs/usr/lib/gcc/x86_64-linux-gnu/7/include-fixed/
  Processing folder (recursive): C:/Users/John Nikko/AppData/Local/Packages/CanonicalGroupLimited.UbuntuonWindows_79rhkp1fndgsc/LocalState/rootfs/usr/include/
  Processing folder (recursive): C:/Users/John Nikko/Desktop/codes/
  Discovering files: 2756 file(s) processed
  0 file(s) removed from database
Attempting to get defaults from compiler in "compilerPath" property: '/usr/bin/gcc'
Done discovering files.
Parsing remaining files...
  Parsing: 0 files(s) processed
Done parsing remaining files.
cpptools/activeDocumentChange
cpptools/textEditorSelectionChange
```
Squiggles do appear in other file like c_cpp_properties.json but not in c/c++ files and headers.

**Comment by sean-mcmanus** [comment]
@bobbrow Any ideas?

**Comment by bobbrow** [comment]
When we split the stdout/stderr streams for debugging formatting errors, this log message was not updated so I can't see the point of failure. :man_facepalming:  I'll fix the message and then we will be able to debug further with the next update.

**Comment by karera123** [comment]
I also installed C/C++ Extension v0.17.5-insider but intellisense still not working and `Microsoft.VSCode.CPP.IntelliSense.Msvc.exe` keeps not running. I'm using Windows 10 Pro April 2018 Update.

**Comment by bobbrow** [comment]
@butchiki the logging for `Failed to spawn IntelliSense process` should be updated in newer versions of the extension.  Are you still hitting this problem and can you see what the error message says now?

**Comment by sean-mcmanus** [comment]
The bug with IntelliSense not working on Windows when the username has a space in it should be fixed with 0.22.0-insiders2: https://github.com/Microsoft/vscode-cpptools/releases .
